tests/integration/integration_python_acceptance_test.go

```diff
@@ -238,8 +247,6 @@ func TestAutomaticVenvCreation(t *testing.T) {
 	// handling by the pulumi CLI itself.
 
 	check := func(t *testing.T, venvPathTemplate string, dir string) {
-		t.Parallel()
-
```

Comment:
Why was this removed?

Reply:
I moved it out of the lambda and into the call sites, eg see:
```
t.Run("RelativePath", func(t *testing.T) {
		t.Parallel()
		check(t, "venv", filepath.Join("python", "venv"))
	})
```

The linter understands this, so it was either do this, or comment all the t.Runs below with `//nolint:paralleltest // check calls t.Parallel()`. I felt this style was nicer.